websites/W/watch.lonelil.com/presence.ts

```diff
@@ -0,0 +1,157 @@
+const presence = new Presence({
+		clientId: "1147910465134002276",
+	}),
+	strings = presence.getStrings({
+		play: "general.playing",
+		pause: "general.paused",
+		viewAMovie: "general.viewAMovie",
+		viewAShow: "general.viewAShow",
+		viewAnime: "general.viewAnime",
+		watchingMovie: "general.watchingMovie",
+		watchingShow: "general.watchingShow",
+		watchingLive: "general.watchingLive",
+		viewingHomePage: "general.viewHome",
+		loading: "kahoot.loadingPage",
+		buttonWatchMovie: "general.buttonWatchMovie",
+		buttonWatchEpisode: "general.buttonViewEpisode",
+		buttonWatchStream: "general.buttonWatchStream",
+		buttonViewMovie: "general.buttonViewMovie",
+		buttonViewShow: "general.buttonViewShow",
+	});
+
+enum logos {
+	logo = "https://i.imgur.com/Niuxx4z.png",
+	home = "https://i.imgur.com/RPr5zwZ.png",
+}
```

Comment:
Enums should be in `PascalCase`
```ts
enum Logos {
  Logo = "...";
  // ...
}
```
For consistency with other presences, you can combine it into the `Assets` enum:
```ts
const enum Assets {
  Logo = "...";
  Home = "...";
}
```